### PR TITLE
Fix #3140: Move message text from R code to C#

### LINKED ISSUE #3140: Move message text from R code to C#
So we can localize them in a regular way. Perhaps pass some form of ID like `#125` similarly to what VS SDK does? Or more readable `#message_id` similar to `Resources.message_id`

### PR DESCRIPTION
**Pull request by int19h**
Add an R helper function to get localizable strings from the IDE, and use it for all user-visible string literals in R code.

This depends on https://github.com/Microsoft/R-Host/pull/109

### CODE REVIEW

File: src/Host/API/Impl/RSessionCallback.cs [hunk 1]
```diff
@@ -51,5 +51,7 @@ public Task<MessageButtons> ShowMessageAsync(string message, MessageButtons butt
         public Task ViewFile(string fileName, string tabName, bool deleteFile, CancellationToken cancellationToken = default(CancellationToken)) => Task.CompletedTask;
         public Task ViewLibraryAsync(CancellationToken cancellationToken = default(CancellationToken)) => Task.CompletedTask;
         public Task ViewObjectAsync(string expression, string title, CancellationToken cancellationToken = default(CancellationToken)) => Task.CompletedTask;
+
+        public string GetLocalizedString(string id) => null;
```
Comment (AlexanderSher) [inline src/Host/API/Impl/RSessionCallback.cs]: Can we provide all those strings at startup in a batch?

Comment (int19h) [inline src/Host/API/Impl/RSessionCallback.cs]: We can, but then it won't reflect culture changes until the session is restarted.

**Comment by MikhailArkhipov** [comment]
👍

**Comment by karthiknadig** [comment]
👍

**Pull request merged**